packages/node-api/integration/Fees.ts

```diff
@@ -1,10 +1,23 @@
 import { setup } from "./Util.js";
 
-export async function fees() {
+export async function storageFees() {
     const { api } = await setup();
     const fee = await api.fees.estimateStorageFee({
         numOfEntries: BigInt(10),
         totSize: BigInt(15000),
     });
     expect(fee).toBeGreaterThanOrEqual(0);
 }
+
+const ONE_LGNT = 1_000_000_000_000_000_000n;
+
+export async function legalFees() {
+    const { api } = await setup();
+    const identityLocFee = await api.fees.estimateLegalFee({ locType: "Identity"});
+    expect(identityLocFee).toEqual(160n * ONE_LGNT);
```

Comment:
You could also use `Currency.nLgnt(...)`.

Reply:
Done.